index.html

```diff
@@ -889,67 +889,133 @@ <h3>Supporting algorithms</h3>
-              |focusedDocument|'s [=origin=], return true.
-            </li>
-            <li>
-              Otherwise, return false.
-            </li>
-          </ol>
-        </li>
-      </ol>
-      <aside class="note">
-        As there might be multiple observers, each with a different [=requested sampling rate=], the underlying
-        [=platform collector=] will need to use a [=sampling rate=] that fulfills all these requirements. This also
-        means that not every data sample from the [=platform collector=] needs to be delivered to each active
-        observer.
-      </aside>
+          </li>
+          <li>
+            [=list/For each=] |origin| in
+            <a href="https://w3c.github.io/picture-in-picture/#initiators-of-active-picture-in-picture-sessions">
+              initiators of active Picture-in-Picture sessions</a>:
+            <ol>
+              <li>
+                If |relevantGlobal|'s [=relevant settings object=]'s [=origin=] is [=same origin=] with |origin|, return true.
+              </li>
+            </ol>
+          </li>
+          <li>
+            If |relevantGlobal|'s [=browsing context=] is [=context is capturing|capturing=], return true.
+          </li>
+          <li>
+            Let |topLevelBC| be |relevantGlobal|'s [=browsing context=]'s [=top-level browsing context=].
+          </li>
+          <li>
+            If |topLevelBC| does not have [=top-level traversable/system focus=], return false.
+          </li>
+          <li>
+            Let |focusedDocument| be the |topLevelBC|'s
+            <a href="https://html.spec.whatwg.org/multipage/interaction.html#currently-focused-area-of-a-top-level-browsing-context">
+              currently focused area</a>'s [=Node/node document=].
+          </li>
+          <li>
+            If |relevantGlobal|'s [=relevant settings object=]'s [=origin=] is [=same origin=] with
+            |focusedDocument|'s [=origin=], return true.
+          </li>
+          <li>
+            Otherwise, return false.
+          </li>
+        </ol>
+      </p>
+      <p>
+        The <dfn>passes privacy test</dfn> steps given the argument |observer:PressureObserver| and
+        its [=relevant global object=] |relevantGlobal|, are as follows:
+        <ol>
+          <li>
+            If |relevantGlobal| is a {{WorkerGlobalScope}} object:
+            <ol>
+              <li>
+                If |relevantGlobal|'s relevant worker is not a <a href="https://html.spec.whatwg.org/multipage/workers.html#active-needed-worker">
+                active needed worker</a>, return false.
+              </li>
+              <li>
+                If |relevantGlobal| is a {{DedicatedWorkerGlobalScope}} object:
+                <ol>
+                  <li>
+                    Let |ownerGlobal| be |relevantGlobal|'s [=owning global object=].
+                  </li>
+                  <li>
+                    Assert: |ownerGlobal| is not null.
```

Comment:
"owning global object" does return null if none of the conditions match, I think you either need to assert there that at least one of the conditions match and remove the "return null" step or react to it here accordingly.

Reply:
Yes, mostly due to following what they did in page life cycle. Yes, let me react to it here.